lite2/reactor/codec.go

```diff
@@ -0,0 +1,13 @@
+package reactor
+
+import (
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/types"
+)
+
+var cdc = amino.NewCodec()
```

Comment:
wonder if it makes sense to switch to proto right away here (cc @marbar3778 )

Reply:
emhh, not right now as there are missing proto types for what is needed here. To get the types you would have to change quite a bit.